scale-tests/configs/2019-10-10-mwt21.env

```diff
@@ -0,0 +1,170 @@
+# Depends on:
+# - TEST_NAME
+# - TEST_S3_BUCKET
+# - TEST_S3_FOLDER
+
+# Workload configuration #######################################################
+#
+# Total CPU quota: 2290
+# Total MEM quota: 4580000
+# Total GPU quota: 40
+
+CLUSTER_URL="https://mwt21.scaletesting.mesosphe.re"
+SECURITY="strict"
+
+DCOS_CLI_URL="https://downloads.dcos.io/cli/releases/binaries/dcos/linux/x86-64/latest/dcos"
+
+# Test configuration ###########################################################
+
+SHOULD_INSTALL_INFRASTRUCTURE=true
+SHOULD_INSTALL_NON_GPU_DISPATCHERS=true
+SHOULD_INSTALL_GPU_DISPATCHERS=false
+SHOULD_RUN_FINITE_STREAMING_JOBS=true
+SHOULD_RUN_INFINITE_STREAMING_JOBS=true
+SHOULD_RUN_BATCH_JOBS=true
+SHOULD_RUN_GPU_BATCH_JOBS=false
+SHOULD_UNINSTALL_INFRASTRUCTURE_AT_THE_END=false
+
+# Infrastructure configuration #################################################
+
+GROUP_NAME="${TEST_NAME}"
+
+SERVICE_NAMES_PREFIX="${TEST_NAME}/"
+INFRASTRUCTURE_OUTPUT_FILE="infrastructure.json"
+
+KAFKA_CLUSTER_COUNT=1
+CASSANDRA_CLUSTER_COUNT=1
+
+ZOOKEEPER_CPUS=10
+ZOOKEEPER_MEM=20000
+ZOOKEEPER_CONFIG='scale-tests/configs/kafka-zookeeper-options.json'
+# Note: empty package repo values will default to latest Universe packages.
+ZOOKEEPER_PACKAGE_REPO="https://universe-converter.mesosphere.com/transform?url=https://infinity-artifacts.s3.amazonaws.com/permanent/confluent-zookeeper/assets/2.6.1-5.1.2e/stub-universe-confluent-zookeeper.json"
+# 2.6.0-5.1.2e from the Universe didn't have role-enforcement support. Used
+# 2.6.1-5.1.2e permanent stub above.
+
+KAFKA_CPUS=10
+KAFKA_MEM=20000
+KAFKA_CONFIG='scale-tests/configs/kafka-options.json'
+# Note: empty package repo values will default to latest Universe packages.
+KAFKA_PACKAGE_REPO="https://universe-converter.mesosphere.com/transform?url=https://infinity-artifacts.s3.amazonaws.com/permanent/confluent-kafka/assets/2.7.1-5.3.0/stub-universe-confluent-kafka.json"
+# 2.7.0-5.3.0 from the Universe didn't have role-enforcement support. Used
+# 2.7.1-5.3.0 permanent stub above.
+
+CASSANDRA_CPUS=10
+CASSANDRA_MEM=20000
+CASSANDRA_CONFIG='scale-tests/configs/cassandra-options.json'
+# Note: empty package repo values will default to latest Universe packages.
+CASSANDRA_PACKAGE_REPO=
+# 2.7.0-3.11.4 from the Universe.
+
+DSENGINE_CPUS=10
+DSENGINE_MEM=20000
+DSENGINE_GPUS=40
+DSENGINE_PACKAGE_REPO=
```

Comment:
Seems like these configs should be below L66?

Reply:
https://github.com/mesosphere/spark-build/pull/552/commits/92c8029652ec6d8ade546c58b576bba8974c1726